rules/windows/privilege_escalation_exploit_cve_202238028.toml

```
@@ -80,8 +82,8 @@ query = '''
 file where host.os.type == "windows" and event.type != "deletion" and
     file.name : "MPDW-constraints.js" and
     file.path : (
-        "?:\\*\\Windows\\system32\\DriVerStoRe\\FiLeRePoSiToRy\\*\\MPDW-constraints.js",
-        "?:\\*\\Windows\\WinSxS\\amd64_microsoft-windows-printing-printtopdf_*\\MPDW-constraints.js"
+        "*\\Windows\\system32\\DriverStore\\FileRepository\\*\\MPDW-constraints.js",
+        "*\\Windows\\WinSxS\\amd64_microsoft-windows-printing-printtopdf_*\\MPDW-constraints.js"
```

to avoid matching on legit path `c:\windows\` : 

```suggestion
        "*?:\\*\\Windows\\system32\\DriverStore\\FileRepository\\*\\MPDW-constraints.js",
        "?:\\*\\Windows\\WinSxS\\amd64_microsoft-windows-printing-printtopdf_*\\MPDW-constraints.js", 
        "\\Device\\HarddiskVolume*\\*\\Windows\\system32\\DriverStore\\FileRepository\\*\\MPDW-constraints.js",
        "\\Device\\HarddiskVolume?\\*\\Windows\\WinSxS\\amd64_microsoft-windows-printing-printtopdf_*\\MPDW-constraints.js"
```